tests/deepprofiler/learning/test_profiling.py

```diff
@@ -120,11 +120,12 @@ def profile(config, dataset):
     return deepprofiler.learning.profiling.Profile(config, dataset)
 
 
-def test_init(config, dataset):
-    prof = deepprofiler.learning.profiling.Profile(config, dataset)
+def test_init(config, dataset, locations):
+    metadata = deepprofiler.dataset.image_dataset.read_dataset(config)
+    prof = deepprofiler.learning.profiling.Profile(config, metadata)
```

Comment:
Why was the dataset object not used here?

Reply:
Fixed.